### DISCUSSION TITLE
希望在入站配置中加入允许互联网访问

### DISCUSSION THREAD

**Discussion by liumingan** [nikkinikki-org/OpenWrt-nikki#101] [Q&A]
用open clash的时候可以远程访问路由器的sock5端口，实现代理。
目前是允许局域网访问，设置验证就多此一举了

**Comment by morytyann** [comment]
直接在防火墙规则里添加一个允许SOCKS5端口入站就可以了

**Reply by liumingan** [reply]
我觉得界面上可以选择的应该是：仅允许局域网访问。
默认情况下局域网访问应该是打开的，互联网访问是关闭的
选择仅允许局域网访问了，就关闭互联网访问。
这个逻辑应该是合理的吧

**Comment by liumingan** [comment]
> 直接在防火墙规则里添加一个允许SOCKS5端口入站就可以了

我在防火墙规则里设置了打开了Sock5端口的入站，可并不能连接上

**Reply by morytyann** [reply]
开启IPv6代理了吗，没有的话开启试试？

**Comment by liumingan** [comment]
Joseph Mory ***@***.***> 于2024年8月22日周四 13:56写道：

> 开启IPv6代理了吗，没有的话开启试试？
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/morytyann/OpenWrt-mihomo/discussions/101#discussioncomment-10415244>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHST53ZU2US62RSVH62IWVLZSV4SBAVCNFSM6AAAAABM4PMGZCVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTANBRGUZDINA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>

ipv6dail 已经开启

**Comment by liumingan** [comment]
ipv6代理已经开启

Liu Mingan ***@***.***> 于2024年8月22日周四 16:05写道：

**Reply by liumingan** [reply]
连接不上是我的错，我把配置文件中的密码写错了。
我没有勾选，允许局域网访问。
在mihomo安卓客户端建立一个socks5的节点连接上了自己的路由器。

**Comment by liumingan** [comment]
不勾选允许局域网访问就无法远程访问内部网络，勾选后可以远程访问内网

**Reply by morytyann** [reply]
这个就是核心的行为了，估计是不允许局域网访问就直接监听127.0.0.1了。

**Comment by liumingan** [comment]
更新到1.7.1后不能访问内部网络了
